XamarinCommunityToolkit/Converters/MultiConverter.shared.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Converters
+{
+    /// <summary>
+    /// Converts an incoming value using all of the incoming converters in sequence.
+    /// </summary>
+    public class MultiConverter : List<IValueConverter>, IValueConverter
+    {
+        /// <summary>
```

Comment:
We should merge https://github.com/xamarin/XamarinCommunityToolkit/pull/139 and then add a property "Parameters" as well